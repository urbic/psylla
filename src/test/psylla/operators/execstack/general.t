{ 1 2 add execstack ? 3 4 sub } exec
