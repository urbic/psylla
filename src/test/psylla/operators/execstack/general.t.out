[sub 4 3 ?]
